### PR TITLE
Add toleration

### PR DESCRIPTION
**Pull request by obeyler**
Proposal to fix the toleration missing for flannel daemonset
Fix issue: https://github.com/jhunt/k8s-boshrelease/issues/68

### CODE REVIEW

**Comment by jhunt** [comment]
Why not just fix the raw spec?

**Comment by obeyler** [comment]
I wanted to be flexible but I agree fix the Flanel could be enought

**Comment by obeyler** [comment]
I change the PR to adapt only flannel

**Comment by jhunt** [comment]
Looks good.  If you can drop the needless whitespace changes I'll merge it.

**Comment by obeyler** [comment]
please point me where are the needless whitespace I don't see them

File: jobs/net-flannel/spec [hunk 1]
```diff
@@ -25,5 +25,4 @@ properties:
   debug:
     default:     false
     description: Enables verbose debugging, suitable for troubleshooting.
-
```
Comment (jhunt) [inline jobs/net-flannel/spec]: This blank line removal is unnecessary and unrelated.

File: jobs/net-weave/spec [hunk 2]
```diff
@@ -24,5 +24,4 @@ properties:
   debug:
     default:     false
     description: Enables verbose debugging, suitable for troubleshooting.
-
```
Comment (jhunt) [inline jobs/net-weave/spec]: This blank line removal is unnecessary and unrelated.

**Review by jhunt — CHANGES_REQUESTED** [review]

File: jobs/net-flannel/spec (hunk 1 above)
Comment (obeyler) [reply jobs/net-flannel/spec]: done

File: jobs/net-weave/spec (hunk 2 above)
Comment (obeyler) [reply jobs/net-weave/spec]: done

**Review by jhunt — APPROVED** [review]
:+1:

**Pull request merged**